package.json

```diff
@@ -35,6 +35,7 @@
       "./setupCoreJsPollyfills.js"
     ],
     "testPathIgnorePatterns": [
+      "<rootDir>/src/components/index.test.tsx",
```

Comment:
This is the test file for typescript bindings and Jest will choke on it if ran. I had it as `index.tests.tsx`, but then found when I tried to run styleguidist through docker, it chokes due to no proper loader. Not sure why it doesn't happen locally, but it made sense to rename it so styleguidist would automatically `ignore` as `.test` files are ignored by default.